--- /Users/zulak/Downloads/pebble-sdk-release-001/sdk/include/pebble_os.h	2013-04-16 17:45:49.000000000 -0700
+++ build/sdk/include/pebble_os.h	2013-04-16 18:43:36.000000000 -0700
@@ -141,7 +141,7 @@
 typedef void *ClickRecognizerRef;
 typedef void (*ClickHandler)(ClickRecognizerRef recognizer, void *context);
 typedef enum {TUPLE_BYTE_ARRAY = 0, TUPLE_CSTRING = 1, TUPLE_UINT = 2, TUPLE_INT = 3} TupleType;
-typedef struct __attribute__((__packed__))
+typedef struct 
 {
   uint32_t key;
   TupleType type : 8;
@@ -181,7 +181,7 @@
     } integer;
   };
 } Tuplet;
-typedef struct __attribute__((__packed__))
+typedef struct 
 {
   uint8_t count;
   Tuple head[];
